http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java

```diff
@@ -56,7 +56,14 @@
 			properties.load(in);
 
 			version = properties.getProperty("runelite.version");
-			rsVersion = Integer.parseInt(properties.getProperty("rs.version"));
+			try
+			{
+				rsVersion = Integer.parseInt(properties.getProperty("rs.version"));
+			}
+			catch (NumberFormatException e)
```

Comment:
Why are you doing this when you have this code already wrapped in try catch? just add another catch clause